archivers/libarchive/Portfile

```diff
@@ -24,6 +25,7 @@ checksums        rmd160  c59dd17f764e33c2984058091ea00b5768e35ce4 \
 homepage         https://libarchive.org/
 master_sites     ${homepage}downloads/
 
+
```

Comment:
Is this exact whitespace? Is it necessary?

Reply:
Guess its just a left over from removing the local blacklist code... If someone really reall objects guess i could remove it...